shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -779,6 +783,93 @@ void onBackPressed() {
     }
   }
 
+  /**
+   * Invoke this from {@link OnBackAnimationCallback#onBackStarted(BackEvent)}.
+   *
+   * <p>This method should be called when the back gesture is initiated, as part of the
+   * implementation of {@link OnBackAnimationCallback}. It's responsible for handling the initial
+   * response to the start of a back gesture, such as initiating animations or preparing the UI for
+   * the back navigation process.
+   *
+   * @param backEvent The BackEvent object containing information about the touch.
+   */
+  @TargetApi(34)
+  @RequiresApi(34)
+  void startBackGesture(@NonNull BackEvent backEvent) {
+    ensureAlive();
+    if (flutterEngine != null) {
+      Log.v(TAG, "Forwarding startBackGesture() to FlutterEngine.");
+      flutterEngine.getNavigationChannel().startBackGesture(backEvent);
```

Comment:
I'm debating if we should create a new channel for this. This looks independent of everything else in the navigation channel, even though it does relate to navigation.

I'll comment more about this on the framework PR.

Reply:
I have added `BackGestureChannel`